src/bundle/Resources/public/scss/_tables.scss

```diff
@@ -5,42 +5,65 @@
     justify-content: space-between;
     background-color: $ez-ground-primary;
     align-items: center;
-    padding: .5rem 1rem;
+    padding: 0 1.25rem;
+    height: 50px;
```

Comment:
I'm not sure if we want to have fixed height of the header, what if the text will have 2 lines? I think `min-height` would be better.